[2025-02-21T15:09:27.721+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-21T15:09:23.027357+00:00 [queued]>
[2025-02-21T15:09:27.764+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-21T15:09:23.027357+00:00 [queued]>
[2025-02-21T15:09:27.764+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-02-21T15:09:27.829+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): kafka_data_stream> on 2025-02-21 15:09:23.027357+00:00
[2025-02-21T15:09:27.854+0000] {standard_task_runner.py:57} INFO - Started process 956 to run task
[2025-02-21T15:09:27.877+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'kafka_spark_dag', 'kafka_data_stream', 'manual__2025-02-21T15:09:23.027357+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/dag_kafka_spark.py', '--cfg-path', '/tmp/tmp4jo1_kjt']
[2025-02-21T15:09:27.881+0000] {standard_task_runner.py:85} INFO - Job 55: Subtask kafka_data_stream
[2025-02-21T15:09:28.152+0000] {task_command.py:416} INFO - Running <TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-21T15:09:23.027357+00:00 [running]> on host df0a4c9b1ab1
[2025-02-21T15:09:28.475+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='kafka_spark_dag' AIRFLOW_CTX_TASK_ID='kafka_data_stream' AIRFLOW_CTX_EXECUTION_DATE='2025-02-21T15:09:23.027357+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-21T15:09:23.027357+00:00'
[2025-02-21T15:09:54.377+0000] {logging_mixin.py:154} INFO - Dataset downloaded successfully.
[2025-02-21T15:09:54.525+0000] {logging_mixin.py:154} INFO - Dataset extracted successfully.
[2025-02-21T15:09:54.526+0000] {logging_mixin.py:154} INFO - No CSV file found in extracted data.
[2025-02-21T15:09:54.533+0000] {logging_mixin.py:154} INFO - Dataset preparation failed.
[2025-02-21T15:09:54.534+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-21T15:09:54.569+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=kafka_spark_dag, task_id=kafka_data_stream, execution_date=20250221T150923, start_date=20250221T150927, end_date=20250221T150954
[2025-02-21T15:09:54.640+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-21T15:09:54.727+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
